cms/static/js/views/settings/grading.js

```diff
@@ -298,10 +303,14 @@ var GradingView = ValidatingView.extend({
         this.descendingCutoffs.push({designation: this.GRADES[gradeLength], cutoff: failBarWidth});
         this.descendingCutoffs[gradeLength - 1]['cutoff'] = Math.round(targetWidth);
 
-        var $newGradeBar = this.gradeCutoffTemplate({ descriptor : this.GRADES[gradeLength],
-            width : targetWidth, removable : true });
+        var newGradeHtmlString = this.gradeCutoffTemplate({
+            descriptor : this.GRADES[gradeLength],
+            width : targetWidth,
+            contenteditable: true,
+            removable : true }).toString();
         var gradeDom = this.$el.find('.grades');
-        gradeDom.children().last().before($newGradeBar);
+
+        gradeDom.children().last().before(newGradeHtmlString);
```

Comment:
Could `newGradeHtmlString` be `newGradeHtml` (the original HTML Snippet) instead, and could this line be refactored to use HtmlUtils.append()?